README.md

```diff
@@ -1,19 +1,131 @@
+and [Seamonkey](https://www.seamonkey-project.org/).
+
+It extends the [Lightning](https://mozilla.org/calendar) add-on to provide
+[Microsoft Exchange](http://microsoft.com/exchange) support.
+
+Exchange Calendar extension can:
+  * Synchronize calendars, tasks, and contacts with a Microsoft Exchange server.
+  * Create, edit and update events and tasks.
+  * Display contacts from your address books and the global address list
+    (they are only readable, they are usable with address autocompletion)
+  * Manage the Exchange « out of office » feature
+
+This extension requires:
+  * Lightning extension corresponding to the Thunderbird release
+  * The Exchange server has to provide an Exchange Web Service
+
+This extension was developed by its original author (Michel Verbraak) for
+Exchange 2007, 2010 and 2013.
+
+The ExchangeCalendar community is now maintaining this extension. Support is
+best effort and cannot be guaranteed. Contributions are welcome.
 
 ## Contributing
 
 This add-on is open source and based on the work of many
 [contributors](https://github.com/ExchangeCalendar/exchangecalendar/graphs/contributors).
 
-### Building
+This project is a community driven effort to keep maintained and uptodate
+the "Exchange EWS Provider" extension created by Michel Verbraak.
+
+Currently, the community is really small and have too few developers to
+give any warranty on the future. Please help us to continue this project.
+
+### Provide feedback, report issue
+
+You are welcome to provide feedback on our github project:
+https://github.com/ExchangeCalendar/exchangecalendar
+
+Please note that the project is built only for latest stable Thunderbird release
+and latest stable Lightning release.
+
+If you use any different release, please write it in your report.
+
+As this add-on extends Thunderbird and Lightning directly, it could be in
+conflict with other add-ons.
+
+If you can give us a list of installed and enabled add-ons, it will help us.
+
+### Get project and build it
+
+To build the add-on, you have to:
+```bash
+# clone this project
+git clone https://github.com/ExchangeCalendar/exchangecalendar.git
+cd exchangecalendar
+# build it
+make build
+```
+Then an `xpi` file is created in the project folder.
+This file is directly installable from the add-on page of Thunderbird.
 
-To compile the add-on, run the `build.sh` script.
+If it doesn't work, please check you have these tools installed (example for
+Debian):
+```bash
+sudo apt install git-core zip
+```
+
+### Modify code
+
+To help us to maintain the code, we will be thankful to follow these guidelines:
+
+* For Javascript files, use tabulations to indent your code
+* For XML files, use spaces
+
+That's a bit strange, we know, but it's like the original code has been written.
+
+As current code indentation is really bad and as we are humans, if you edit a
+file, please first use formatters to automatically have better code indentation:
```

Comment:
```diff
+As current code indentation is really bad and as we are humans, if you edit a
+file, please first use formatters to automatically have better code indentation:
```
I think that's not correctly translate what I mean. What do you think about the following sentence to replace this one?

> The code from the original project is not very well indented and its code style is not consistent. So, when you begin to work on a code that ExchangeCalendar community has not modified recently (you can know this either by looking for last commits date (before August 2017) or by looking the overall indentation and code style), please first use a code beautifier/formatter and commit directly modifications with a message like "auto formatting". Also, with new code, we will be grateful if you can parse your code to such a tool too before sending your patch.